test/suites/container_devices_disk.sh

```diff
@@ -59,3 +60,28 @@ test_container_devices_disk_shift() {
   lxc storage volume delete "${POOL}" foo-shift
   lxc stop foo -f
 }
+
+test_container_devices_raw_mount_options() {
+  configure_loop_device loop_file_1 loop_device_1
+  mkfs.vfat "${loop_device_1}"
```

Comment:
Needs a `# shellcheck disable=SC2154` comment above is so shellcheck doesn't freak out.

Reply:
The comment must go above the mkfs.vfat line, not the configure_loop_device.

As far as I can tell, that's the only issue left before we can merge this.

Reply:
Updated